packages/manager/modules/exchange/src/domain/srv-autoconfig/domain-srv-autoconfig.controller.js

```diff
@@ -21,14 +21,43 @@ export default class ExchangeDomainSrvAutoconfigCtrl {
 
     this.$routerParams = wucExchange.getParams();
     this.domain = navigation.currentActionData;
+    this.domainDiag = {};
```

Comment:
```suggestion
    this.domainDialog = {};
```

Reply:
domainDiag was not my choice, was already defined like that before.
For info, it's not mean Dialog but Diagnose ;)